<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listas de Compras</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="fr_superior">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="bt_articulos">
         <property name="text">
          <string>Artículos</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>3</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bt_nuevoarticulo">
         <property name="text">
          <string>Nuevo Artículo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>3</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bt_editararticulo">
         <property name="text">
          <string>Editar Artículo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>3</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bt_borrararticulo">
         <property name="text">
          <string>Borrar Artículo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fr_inferior">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 55, 235);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArticulos">
    <property name="title">
     <string>Articulos</string>
    </property>
    <addaction name="action_articulos"/>
    <addaction name="action_editararticulo"/>
    <addaction name="action_nuevoarticulo"/>
    <addaction name="action_borrararticulo"/>
   </widget>
   <addaction name="menuArticulos"/>
  </widget>
  <action name="actionddd">
   <property name="text">
    <string>ddd</string>
   </property>
  </action>
  <action name="actionNuevo_Articulo">
   <property name="text">
    <string>Nuevo Artículo</string>
   </property>
  </action>
  <action name="actionEditar_Articulo">
   <property name="text">
    <string>Editar Artículo</string>
   </property>
  </action>
  <action name="actionBorrar_Articulo">
   <property name="text">
    <string>Borrar Artículo</string>
   </property>
  </action>
  <action name="actionArticulos_2">
   <property name="text">
    <string>Artículos</string>
   </property>
  </action>
  <action name="action_editararticulo">
   <property name="text">
    <string>Editar Artículo</string>
   </property>
  </action>
  <action name="action_borrararticulo">
   <property name="text">
    <string>Borrar Artículo</string>
   </property>
  </action>
  <action name="actiondd">
   <property name="text">
    <string>dd</string>
   </property>
  </action>
  <action name="action_articulos">
   <property name="text">
    <string>Artículos</string>
   </property>
  </action>
  <action name="action_nuevoarticulo">
   <property name="text">
    <string>Nuevo Artículo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
